### DISCUSSION TITLE
Cross platform

### DISCUSSION THREAD

**Discussion by hermannfischer** [hjoykim/THREE#29] [Q&A]
Does anyone know, if I can use THREE on Linux and Mac?

**Comment by hjoykim** [comment]
hello

On Linux or Mac,
Referring to an example using OpenTK,(https://github.com/opentk/LearnOpenTK)
I think you can use THREE to create the app that you want.

You probably need to retrieve the IGraphicContext from your Application to call THREE.Renderer,
Some Camera Controllers may require modifications.

If you use Windows 10/11, using WSL may be a good starting point.
